client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx

```diff
@@ -106,7 +106,8 @@ const BrewItem = createClass({
 			<div className='info'>
 				{brew.tags ? <>
 					<span className='brewTags' title={`Tags:\n${brew.tags.join('\n')}`}>
-						<i className='fas fa-tags'/> {brew.tags.join(', ')}
+						<i className='fas fa-tags'/>
+						{brew.tags.map((tag, idx)=>{return <><span>{tag}</span>{'\n'}</>;})}
```

Comment:
Really should do something about the prefixed tags though..
https://homebrewery-stage.herokuapp.com/user/erics
<img width="341" alt="image" src="https://user-images.githubusercontent.com/2605467/188547762-c7be52e7-9628-431c-b213-33cb411ff86b.png">

Change to:
```
{brew.tags.map( (tag) => { 
    return (parts = tag.match(/(.+):(.+)/)) ? <><span class="{parts[1]}">{parts[2]}</span></> : <><span>{tag}</span></>;
    }
)}
```
(JS logic tested, React coding not tested; adjust code style to suit)

Reply:
Added this.